miniapps/tools/check-1st-ti.cpp

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
I am not sure we want this here... can it be a check instead?

Reply:
This has already been converted to a unit check, in test_ode2.ccp.
The check allows to visualize a time trace of the dummy problem, so it is not fully redundant, put purpose is faily minimal. I have not strong feelings either way.

Reply:
I vote to remove the two checks